module/VuFind/src/VuFind/Hierarchy/TreeRenderer/PluginManager.php

```diff
@@ -38,6 +38,25 @@
  */
 class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
 {
+    /**
+     * Default plugin aliases.
+     *
+     * @var array
+     */
+    protected $aliases = [
+        'jstree' => 'VuFind\Hierarchy\TreeRenderer\JSTree'
+    ];
+
+    /**
+     * Default plugin factories.
+     *
+     * @var array
+     */
+    protected $factories = [
+        'VuFind\Hierarchy\TreeRenderer\JSTree' =>
+            'VuFind\Hierarchy\TreeRenderer\Factory::getJSTree'
```

Comment:
Might be worth replacing this static factory with a non-static one (i.e. JSTreeFactory, implementing FactoryInterface).